core/src/bin/run.ts

```diff
@@ -56,11 +56,18 @@ export class RunCLI extends CLI {
     GrouparooCLI.logCLI(this.name, false);
     this.checkWorkers();
 
+    const scheduleCount = await Schedule.count();
+    if (scheduleCount === 0) {
```

Comment:
Ideally, this would be after `main()`, so the new config files would be loaded in first before checking.  As is (as you call out), you would need to `grouparoo apply` the first time or whenever you add your first schedule